.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+node_js:
+- node
+script:
+- npm test
+- npm run build
```

Comment:
I've looked through a lot FE peeps existing travis configs and they don't seem to have any scripts (no test or build). I thought we should add testing before deployment but is this not best practice with a react app?

Reply:
This looks reasonable for now! Note that I'm using the yarn instead of npm. So you may want to stick with that.